modules/service/src/main/scala/lucuma/odb/service/UserInvitationService.scala

```diff
@@ -53,8 +58,35 @@ object UserInvitationService:
     def apply(id: UserInvitation.Id, detail: Option[String]): OdbError.InvitationError =
       OdbError.InvitationError(UserInvitation.Id.fromString.reverseGet(id), detail)
 
-  def instantiate[F[_]: MonadCancelThrow](using Services[F]): UserInvitationService[F] =
+  def instantiate[F[_]: MonadCancelThrow](emailConfig: Config.Email, httpClient: Client[F])(using Services[F]): UserInvitationService[F] =
     new UserInvitationService[F]:
+      
+      // TODO: move to UserInvitation class?
+      extension(invitation: UserInvitation)
+        def token: String = s"${invitation.id}.${invitation.body}"
+
+      def sendInvitation(input: CreateUserInvitationInput, invitation: UserInvitation)(
```

Comment:
Really nice. I wonder if we should (eventually) add  the ability to easily change the contents of the email. I can foresee changes being requested often

Reply:
Right. I should have commented on that. These messages are `VERY` temporary. We need a better mechanism for creating and editing them.